ui/app/AppLayouts/Wallet/HistoryTab.qml

```diff
@@ -205,10 +217,39 @@ Item {
     }
 
     ListView {
+        id: transactionListRoot
         anchors.topMargin: 20
         anchors.fill: parent
+        anchors.bottomMargin: 69
+        anchors.bottom: extraButtons.top
         model: walletModel.transactions
         delegate: transactionListItemCmp
+        ScrollBar.vertical: ScrollBar {
+            id: scrollBar
+        }
+    }
+
+    RowLayout {
+        id: extraButtons
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.bottom: parent.bottom
+        anchors.top: transactionListRoot.bottom
```

Comment:
This creates a loop, since the bottom of `transactionListRoot` depends on `extraButtons` top.

What you can do instead is remove the top anchor and just set the `height` of `extraButtons`.  Something like:

```qml
height: loadMoreButton.height
```